python/tester/upload.py

```diff
@@ -56,12 +56,14 @@
 class Instrument:
     n_snaps: int
     n_detectors: int
+    sal_index: int = 0
```

Comment:
Good idea, given that (for better or worse) we're already assuming this in fan-out service.